src/components/Equity/Equity.tsx

```diff
@@ -160,11 +170,13 @@ export const Equity = ({ assetId, accountId }: EquityProps) => {
           {renderUnderlyingAssets}
         </Flex>
       </CardHeader>
-      <CardBody pt={0} pb={2}>
-        <Stack spacing={0} mt={2} mx={-4} divider={stackDivider}>
-          {renderEquityRows}
-        </Stack>
-      </CardBody>
+      {shouldDisplayEquityRows && (
```

Comment:
It would show some empty spaces, avoid the node to be rendered if there are no rows looks like a better solution than toggling styles so I would keep it like this